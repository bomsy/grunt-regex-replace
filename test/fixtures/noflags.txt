Keep [FIELD] Expect [field] Expect [field] noflags.txt
Keep [OTHER] Expect [other123] Expect [other456]