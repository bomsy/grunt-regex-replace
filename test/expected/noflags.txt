Keep [FIELD] Expect [replaced] Expect [replaced] noflags.txt
Keep [OTHER] Expect [ok123] Expect [ok456]